--- mysql-5.6.13/sql/item_func.h	2013-07-10 12:17:27.000000000 -0400
+++ mysql-5.6.13/sql/item_func.h.fix	2013-09-20 18:00:14.186270995 -0400
@@ -228,7 +228,7 @@
                      void * arg, traverse_order order);
   inline double fix_result(double value)
   {
-    if (isfinite(value))
+    if (std::isfinite(value))
       return value;
     null_value=1;
     return 0.0;
@@ -262,7 +262,7 @@
   */
   inline double check_float_overflow(double value)
   {
-    return isfinite(value) ? value : raise_float_overflow();
+    return std::isfinite(value) ? value : raise_float_overflow();
   }
   /**
     Throw an error if the input BIGINT value represented by the
--- mysql-5.6.13/sql/field.cc	2013-07-10 12:17:27.000000000 -0400
+++ mysql-5.6.13/sql/field.cc.fix	2013-09-20 18:01:01.474598654 -0400
@@ -2348,7 +2348,7 @@
     return TYPE_WARN_OUT_OF_RANGE;
   }
   
-  if (!isfinite(nr)) // Handle infinity as special case
+  if (!std::isfinite(nr)) // Handle infinity as special case
   {
     overflow(nr < 0.0);
     return TYPE_WARN_OUT_OF_RANGE;
